idtitlecreatedAtclosedAtclosed
MDU6SXNzdWUzMDY3NzY2OTI=Places to be corrected2018-03-20T08:56:22Z2018-03-21T13:10:36ZTrue
MDU6SXNzdWUzMDc0OTY5MTY=Ambiguity ？2018-03-22T03:50:56Z2018-03-23T00:24:26ZTrue
MDU6SXNzdWUzMDc1MDgyNDU=Mistakes?2018-03-22T05:10:42Z2018-03-22T13:43:40ZTrue
MDU6SXNzdWUzMDc4NzY5ODM=Mistake！2018-03-23T02:13:30Z2018-03-23T20:40:08ZTrue
MDU6SXNzdWUzMDk1Nzk4Mjc=Mistake2018-03-29T00:37:16Z2018-03-29T13:48:38ZTrue
MDU6SXNzdWUzMTA0MDIxODI=Limitation2018-04-02T05:52:54Z2018-04-02T09:17:47ZTrue
MDU6SXNzdWUzMTA3Mzc2MDg=Difficult to understand2018-04-03T08:18:01Z2018-04-03T13:09:13ZTrue
MDU6SXNzdWUzMTIzOTQ2Mjc=Unable to be represented2018-04-09T05:05:24Z2018-07-09T12:46:03ZTrue
MDU6SXNzdWUzMTI3NjUzOTI=Unable to understand2018-04-10T04:02:38Z2018-04-11T14:11:09ZTrue
MDU6SXNzdWUzNjI2NzQ5ODU=mislabeled variable 2018-09-21T15:49:28Z2018-09-22T17:23:18ZTrue
MDU6SXNzdWUzNjM5MzkyMDU=download.py cannot pull the image files2018-09-26T09:37:26Z2019-03-29T14:38:05ZTrue
MDU6SXNzdWU0MjkyMjQ2MzU=Pandoc 2.0: RuntimeError '--latex-engine' has been removed2019-04-04T11:23:46Z2019-05-05T14:58:20ZTrue
MDU6SXNzdWU0MzE0NDY0MDY=maybe there are something wrong in your note2019-04-10T11:18:14Z2019-05-05T14:50:01ZTrue
MDU6SXNzdWU0ODcxNDE2MjQ=Momentum formulation2019-08-29T19:41:38ZFalse
